doc/classes/String.xml

```diff
@@ -549,78 +620,80 @@
+				Formats the string representing a number to have an exact number of [param digits] [i]after[/i] the decimal point.
 			</description>
 		</method>
 		<method name="pad_zeros" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="digits" type="int" />
 			<description>
-				Formats a number to have an exact number of [param digits] before the decimal point.
+				Formats the string representing a number to have an exact number of [param digits] [i]before[/i] the decimal point.
 			</description>
 		</method>
 		<method name="path_join" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="file" type="String" />
 			<description>
-				If the string is a path, this concatenates [param file] at the end of the string as a subpath. E.g. [code]"this/is".path_join("path") == "this/is/path"[/code].
+				If the string is a file path, this concatenates [param file] at the end of the string as a subpath. E.g. [code]"this/is".path_join("path") == "this/is/path"[/code].
 			</description>
 		</method>
 		<method name="repeat" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="count" type="int" />
 			<description>
-				Returns original string repeated a number of times. The number of repetitions is given by the argument.
+				Repeats this string a number of times. [param count] needs to be greater than [code]0[/code]. Otherwise, returns an empty string.
 			</description>
 		</method>
 		<method name="replace" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="forwhat" type="String" />
 			<description>
-				Replaces occurrences of a case-sensitive substring with the given one inside the string.
+				Replaces all occurrences of [param what] inside the string with the given [param forwhat].
 			</description>
 		</method>
 		<method name="replacen" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="forwhat" type="String" />
 			<description>
-				Replaces occurrences of a case-insensitive substring with the given one inside the string.
+				Replaces all [b]case-insensitive[/b] occurrences of [param what] inside the string with the given [param forwhat].
 			</description>
 		</method>
 		<method name="rfind" qualifiers="const">
 			<return type="int" />
 			<param index="0" name="what" type="String" />
 			<param index="1" name="from" type="int" default="-1" />
 			<description>
-				Returns the index of the [b]last[/b] case-sensitive occurrence of the specified string in this instance, or [code]-1[/code]. Optionally, the starting search index can be specified, continuing to the beginning of the string.
+				Returns the index of the [b]last[/b] occurrence of the specified string in this instance, or [code]-1[/code] if there are none. The starting search index can be specified with [param from], continuing to the beginning of the string.
```

Comment:
Don't forget to borrow all improvements from the main `find()` description to `rfind()` and `rfindn()`